index.yml

```diff
@@ -197,6 +197,12 @@ wsh57:
 webview2:
   Description: Microsoft Edge Web View 2
   Category: Essentials
+powershell:
+  Description: PowerShell Wrapper For Wine
+  Category: Essentials
+powershell_core:
+  Description: Microsoft PowerShell
```

Comment:
Minor nit: Make sure to specify that this is PowerShell Core, there's some small diferences between this and Windows PowerShell :smile: